test/io/db_config.sql

```diff
@@ -6,6 +6,7 @@ ALTER ROLE db_config_authenticator SET pgrst.openapi_server_proxy_uri = 'https:/
 ALTER ROLE db_config_authenticator SET pgrst.jwt_secret = 'REALLY=REALLY=REALLY=REALLY=VERY=SAFE';
 ALTER ROLE db_config_authenticator SET pgrst.jwt_secret_is_base64 = 'false';
 ALTER ROLE db_config_authenticator SET pgrst.jwt_role_claim_key = '."a"."role"';
+ALTER ROLE db_config_authenticator SET pgrst.jwt_cache_max_lifetime = '3600';
```

Comment:
How about a different value to ensure the in-db config works?

Reply:
Ah, but the default value is `0`. That's why I set `3600` and `7200` for different authenticators.

Reply:
No, I mean these values are the same as the `no-defaults-*.config` . Try removing this line, the io tests will still pass right?

Reply:
Hmm... It actually fails, it gives me this error:

```log
E           jwt-secret-is-base64 = false
E         - jwt-cache-max-lifetime = 3600
E         ?                          ^
E         + jwt-cache-max-lifetime = 86400
E         ?                          ^ +
E           log-level = "info"
```

Because the `config/no-defaults.config` has `86400` and the `config/expected/no-defaults-with-db.config` has `3600`. The same happens with the other authenticator.

Reply:
My bad, I didn't see that the `expected` configs changed. It's all good.